wagtail/contrib/search_promotions/forms.py

```diff
@@ -45,12 +61,29 @@ class Meta:
 
 
 class SearchPromotionsFormSet(SearchPromotionsFormSetBase):
-    minimum_forms = 1
-    minimum_forms_message = _(
+    """
+    Form set for search promotions
+
+    Attributes
+
+        - minimum_forms (int): Number of forms required for this set.
+    """
+
+    REQUIRED_FIELDS_MISSING = _("You must recommend a related page OR an external url.")
+    TOO_MANY_FIELDS = _("Please only select a Page OR enter an external url.")
+    MINIMUM_FORMS_MESSAGE = _(
```

Comment:
To avoid noise in the PR, can you please update these message properties to be `lower_camel_case`, not critical but unless there is a need to change the existing ones we should try to stick with the convention in the file.